src/components/client/core/Drawer/Drawer.stories.tsx

```diff
@@ -50,6 +51,13 @@ export const WithContext: Story = {
   ),
 };
 
+export const OpenState: Story = {
+  ...WithContext,
+  play: openState,
+  name: "[TEST] Open State",
+  tags: ["test"],
```

Comment:
We can do whatever we want here, in terms of naming the tests, naming the stories, and so on; this is just an example. I figured it would be useful to have some convention that separates "test" stories from regular stories, since when a user clicks a story like this, it will automatically trigger interaction in the test which might be jarring or confusing.

Related note: the tags can be wrapped in an extracted enum as we expand. Tagging is probably a good practice that will help us with refactors (let alone the built-in Storybook search functionality optimizations).